src/taipy/core/__init__.py

```diff
@@ -11,16 +11,17 @@
 
 """# Taipy Core
 
-The Taipy Core package is a Python library designed to build powerful and customized data-driven back-end
-applications. It provides the necessary tools to help Python developers transform their algorithms into a
+The Taipy Core package is a Python library designed to build powerful, customized, data-driven back-end
+applications. It provides the tools to help Python developers transform their algorithms into a
 complete back-end application.
 More details on the [Taipy Core](../../../manuals/core/) functionalities are available in the user manual.
 
-To build a Taipy Core application, the first step consists of setting up the Taipy configuration to design the
-characteristics and the behavior of your application.
+To build a Taipy Core application, the first step consists of setting up the Taipy configuration to design
+your application's characteristics and behaviors.
 Import `Config^` from the `taipy.config^` module, then use the various methods of the `Config^` singleton class to
-configure your core application. In particular configure the [data nodes](../../../manuals/core/config/data-node-config,
-[tasks](../../../manuals/core/config/task-config), and [scenarios](../../../manuals/core/config/scenario-config).
+configure your core application. In particular, configure the
+[data nodes](../../../manuals/core/config/data-node-config), [tasks](../../../manuals/core/config/task-config),
```

Comment:
what about the pipeline? :D

Reply:
I am starting to progressively make the pipeline disappear from the main concepts in the documentation. 

This is a big change we are thinking about with Vincent.
We would like not to have to configure the pipeline directly at the config level anymore. But just an attribute of a scenario config. The purpose in the long term is to stop considering the pipeline as an entity but just as an attribute of a scenario. 


Something like:
Instead of having the following
```
pipeline_1_cfg = Config.configure_pipeline("training", [task_1_cfg, task_2_cfg])
pipeline_2_cfg = Config.configure_pipeline("scoring", [task_1_cfg, task_2_cfg])

scenario_cfg = Config.configure_scenario(
        id="my_sc_id", 
        pipelines=[pipeline_1_cfg, pipeline_2_cfg ], 
        comparators = {}
        frequency = MONTHLY
)
```
We would like to have this:
```
scenario_cfg = Config.configure_scenario(
        id="my_sc_id", 
        tasks=[task_1_cfg, task_2_cfg, tack_2_cfg], 
        comparators = {}
        frequency = MONTHLY
)
scenario_cfg.add_pipeline("training", task_1_cfg, task_2_cfg)
scenario_cfg.add_pipeline("scoring", task_1_cfg, task_2_cfg)
```

Reply:
Does it make sense ?

Reply:
I agree.

In any case, this is for now just a general idea. Nothing is designed yet.  I guess a good API design should be the first and most important thing to do.

Thx,